prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-documents/adjudicator-documents.component.html

```diff
@@ -48,6 +48,10 @@ <h2 class="mb-2">Documents</h2>
                 <mat-icon class="mr-1">attachment</mat-icon>
                 {{ document.filename | default }}
               </button>
+              <button mat-button
```

Comment:
You will want to use the material icon button instead of the regular button.  It provides a border radius, which currently when you mouse over the button is show as rectangular.  You'll also want to add a bit of space between the buttons, and I would add that to the attachment button likely `mr-2` would provide sufficient space.

Reply:
Save you looking it up the directive is `mat-icon-button`

Reply:
> You will want to use the material icon button instead of the regular button. It provides a border radius, which currently when you mouse over the button is show as rectangular. You'll also want to add a bit of space between the buttons, and I would add that to the attachment button likely `mr-2` would provide sufficient space.

Yep, mr-2 is just perfect. Changes made.